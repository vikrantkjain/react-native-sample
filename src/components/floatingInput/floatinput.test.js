import {fireEvent, render, screen} from '@testing-library/react-native';
import FloatingInput from './index';

describe('Test for FloatingInput', () => {
  const onChangeTextMock = jest.fn();
  const changeText = 'Hello';
  const tree = render(<FloatingInput onChangeText={onChangeTextMock} />);
  it('Render successfully', () => {
    expect(tree).toBeDefined();
  });
  it('Render correctly', () => {
    const treeToJSON = tree.toJSON();
    expect(treeToJSON).toMatchSnapshot();
  });
  fireEvent.changeText(screen.getByTestId('Floating-input'), changeText);
});
