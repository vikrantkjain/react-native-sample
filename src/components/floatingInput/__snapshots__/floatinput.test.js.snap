// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for FloatingInput Render correctly 1`] = `null`;
